config/sync/user.role.content_administrator.yml

```diff
@@ -14,6 +14,9 @@ permissions:
   - 'access toolbar'
   - 'administer account settings'
   - 'administer menu'
+  - 'administer site_setting_entity display'
+  - 'administer site_setting_entity fields'
+  - 'administer site_setting_entity form display'
```

Comment:
These permissions are wrong, remove them and ensure the site admin and the content editor can visit the site settings overview page and what else is necessary. Test your work please.

Reply:
Okay, done.